content/02.main-text.md

```diff
@@ -2,34 +2,31 @@
 
 Openness in research – which includes sharing code, data, and manuscripts – benefits the researchers who practice it [@doi:10.7554/eLife.16800], their scientific peers, and the public.
 `TODO: more references needed`
-Traditionally however, review articles, which aim to present the state of the art in a scientific field, are written in private by a single research group or a small team of colleagues.
-In contrast, broadly opening the process to anyone engaged in the topic --- such that planning, organizing, writing, and editing occur collaboratively in a public forum where anyone is welcome to participate --- can maximize a review's value.
-Open drafting of reviews can be especially helpful for rapidly developing fields where integrating diverse opinions and expertise can best capture state-of-the-art knowledge.
+However, though review articles aim to present the state of the art in a scientific field, they are written in private by a single research group or a small team of colleagues.
+In contrast, broadly opening the process to anyone engaged in the topic --- such that planning, organizing, writing, and editing occur collaboratively in a public forum where anyone is welcome to participate --- may maximize a review's value.
+Open drafting of reviews may be especially helpful for capturing state-of-the-art knowledge about rapidly advancing research topics at the intersection of existing disciplines where contributors bring diverse opinions and expertise.
 
-Therefore, in August 2016 we embarked on the "Deep Review" to survey deep learning in precision medicine.
+In August 2016 we identified the role that deep learning was beginning to play in biomedical research as one such area, and we started an open online effort to survey deep learning's role in precision medicine and to predict its effect in the future.
 In May 2017, the project released a complete review titled "Opportunities and obstacles for deep learning in biology and medicine" [@doi:10.1101/142760].
-Subsequently, we have continued to maintain the project and accept new contributions as the manuscript undergoes traditional peer review at a journal.
-Here we'll discuss our experience leading the Deep Review, including the pros and cons of open collaborative reviews, as well as the infrastructure we created, termed Manubot, to enable open manuscript writing online.
+While the article was under review, we continued to maintain the project and accepted new contributions.
+In January 2018, the article was accepted by the journal that it was initially submitted to.
```

Comment:
The provisional acceptance email was February 2018, right?

Reply:
Feb 28! 👍  Changed